selftests/unit/test_astring.py

```diff
@@ -18,19 +18,20 @@ def testTabularOutput(self):
                           '/bin/bar/sbrubles /home/myuser/sbrubles'))
 
     def testTabularWithConsoleCodes(self):
-        matrix = [("a", "bb", "ccc", "dddd", "last"),
+        matrix = [("a", "an", "dog", "word", "final"),
                   ("\x1b[94ma",             # {BLUE}a
-                   "\033[0mbb",             # {END}bb
+                   "\033[0man",             # {END}an
                    "cc\033[91mc",   # cc{RED}c
-                   # {RED}d{GREEN}d{BLUE}d{GREY}d{END}
+                   # {RED}d{GREEN}d{BLUE}d{GRAY}d{END}
```

Comment:
Grey it's valid English, the problem is that we're using en_US dict, so "gray" is the proper spelling for that variation.